substrate/frame/ranked-collective/src/benchmarking.rs

```diff
@@ -126,7 +126,7 @@ benchmarks_instance_pallet! {
 	}
 
 	vote {
-		let class = T::Polls::classes().into_iter().next().unwrap();
+		let class = T::Polls::classes().into_iter().next().ok_or(BenchmarkError::Skip)?;
```

Comment:
what weight `Skip` will result?

Reply:
The Secretary Collective requires skipping these extrinsics to generate weights for the ranked_collective pallet instance which uses this NoOp implementation.

Reply:
Yeah I have not checked the actual code, what we need to do here if `classes()` returns nothing, we need to pass some invalid poll index. The function is still trying to access the storage etc.